framework/src/media/audio/audio_manager.c

```diff
@@ -650,7 +727,8 @@ audio_manager_result_t destroy_audio_stream_in(void)
 	if (g_audio_in_cards[g_actual_audio_in_card_id].status == AUDIO_CARD_RUNNING) {
 		g_audio_in_cards[g_actual_audio_in_card_id].status = AUDIO_CARD_READY;
 		g_audio_in_cards[g_actual_audio_in_card_id].device_id = -1;
-		g_audio_in_cards[g_actual_audio_in_card_id].config.volume = 0;
+		g_audio_in_cards[g_actual_audio_in_card_id].config.cur_volume = 0;
```

Comment:
we will remove destroy, and use reset only right?
And, reset means clean the allocated resource not related to h/w
I mean device_id & volume is tightly coupled with h/w, if we reset all values here,
How we can set again?
So clean buffer & close pcm are enough.